server/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:0.10
+
+RUN \
+  useradd -d /app -m oam
+
+USER oam
+ENV HOME /app
+WORKDIR /app
+
+ADD package.json /app/
+
+RUN \
+  npm install
+
+ADD . /app/
+
+VOLUME ["/app"]
+EXPOSE 8000
```

Comment:
I'd add an `ENV PORT 8000` to match the `EXPOSE` directive.

It'd also be convenient to add `CMD npm start` so that plain old `docker run` will start up the server with suitable defaults.